java/test/jmri/jmrit/display/layoutEditor/LayoutEditorDialogs/LayoutTrackEditorsTest.java

```diff
@@ -48,6 +50,13 @@ public void testCTor() {
         Assert.assertNotNull("exists", layoutTrackEditors);
     }
 
+    @Test
+    public void testBundleStrings() {
```

Comment:
You removed this from one file and added it here too.  It should also be in the bundle test.

Reply:
Removing…